var/spack/repos/spack_repo/builtin/packages/w3nco/package.py

```diff
@@ -30,6 +30,7 @@ def flag_handler(self, name, flags):
                 self.spec.satisfies("%oneapi")
                 or self.spec.satisfies("%apple-clang")
                 or self.spec.satisfies("%clang")
+                or self.spec.satisfies("%gcc@14")
```

Comment:
Presumably applies going forward:
```suggestion
                or self.spec.satisfies("%gcc@14:")
```

Reply:
Whoops. Yeah. I had it the right way in my "real" packages location (the new fun repo), but typed it wrong in the "old" packages location in the spack clone.